### ISSUE TITLE
Version 2.0 breaks thermostat card

### ISSUE THREAD

**Issue by kmlucy** [thomasloven/lovelace-layout-card#37]
When I update from legacy to 2.0, My thermostat card breaks as shown:

![image](https://user-images.githubusercontent.com/13952475/68631725-d0659300-04b9-11ea-9472-c58cd2998534.png)

```
panel: true
cards:
  - type: custom:layout-card
    layout: horizontal
    max_columns: 3
    cards:
      - type: thermostat
        entity: climate.hallway
```

**Comment by raulvasquez** [comment]
I can confirm. Running into the same issue.

**Comment by Bluhme1** [comment]
Same here

**Comment by thomasloven** [comment]
Thermostat card in itself is broken.
Fix is coming up
https://github.com/home-assistant/home-assistant-polymer/pull/4172

**Comment by Bluhme1** [comment]
Hi Thomas
But the Thermostat card works in other views. Only a problem when used with Layout Card

**Comment by SeLLeRoNe** [comment]
Same here since last layout-card update :)

Side note, it seems to be working fine on mobile

**Comment by SeLLeRoNe** [comment]
As additional information I have noticed that sometimes all the cards con on a single column without "breaking" into columns, with a refresh without cache fix it, but it looks quite random and I cannot see errors in the Console

**Comment by kmlucy** [comment]
Fixed with 0.102.
